classes/citation/CitationDAO.php

```diff
@@ -1,253 +0,0 @@
-    {
-        $seq = $citation->getSequence();
-        if (!(is_numeric($seq) && $seq > 0)) {
-            // Find the latest sequence number
-            $result = $this->retrieve(
-                'SELECT MAX(seq) AS lastseq FROM citations
-				WHERE publication_id = ?',
-                [(int)$citation->getData('publicationId')]
-            );
-            $row = $result->current();
-            $citation->setSequence($row ? $row->lastseq + 1 : 1);
-        }
-
-        $this->update(
-            sprintf('INSERT INTO citations
-				(publication_id, raw_citation, seq)
-				VALUES
-				(?, ?, ?)'),
-            [
-                (int) $citation->getData('publicationId'),
-                $citation->getRawCitation(),
-                (int) $seq
-            ]
-        );
-        $citation->setId($this->getInsertId());
-        $this->_updateObjectMetadata($citation);
-        return $citation->getId();
-    }
-
-    /**
-     * Retrieve a citation by id.
-     *
-     * @param int $citationId
-     *
-     * @return ?Citation
-     */
-    public function getById($citationId)
-    {
-        $result = $this->retrieve(
-            'SELECT * FROM citations WHERE citation_id = ?',
-            [$citationId]
-        );
-        $row = $result->current();
-        return $row ? $this->_fromRow((array) $row) : null;
-    }
-
-    /**
-     * Import citations from a raw citation list of the particular publication.
-     *
-     * @param int $publicationId
-     * @param string $rawCitationList
-     *
-     * @hook CitationDAO::afterImportCitations [[$publicationId, $existingCitations, $importedCitations]]
-     */
-    public function importCitations($publicationId, $rawCitationList)
-    {
-        assert(is_numeric($publicationId));
-        $publicationId = (int) $publicationId;
-
-        $existingCitations = $this->getByPublicationId($publicationId)->toAssociativeArray();
-
-        // Remove existing citations.
-        $this->deleteByPublicationId($publicationId);
-
-        // Tokenize raw citations
-        $citationTokenizer = new CitationListTokenizerFilter();
-        $citationStrings = $citationTokenizer->execute($rawCitationList);
-
-        // Instantiate and persist citations
-        $importedCitations = [];
-        if (is_array($citationStrings)) {
-            foreach ($citationStrings as $seq => $citationString) {
-                if (!empty(trim($citationString))) {
-                    $citation = new Citation($citationString);
-                    // Set the publication
-                    $citation->setData('publicationId', $publicationId);
-                    // Set the counter
-                    $citation->setSequence($seq + 1);
-
-                    $this->insertObject($citation);
-
-                    $importedCitations[] = $citation;
-                }
-            }
-        }
-
-        Hook::call('CitationDAO::afterImportCitations', [$publicationId, $existingCitations, $importedCitations]);
```

Comment:
Can you please add this hook removal to the list here https://github.com/pkp/pkp-lib/blob/main/classes/core/PKPApplication.php#L173